scipp-imaging-2D.ipynb

```diff
@@ -224,218 +162,516 @@
    },
    "outputs": [],
    "source": [
-    "# Load the data into the dataset\n",
-    "ds[\"sample\"] = tiffs_to_variable(sample_path)\n",
-    "ds[\"reference\"] = tiffs_to_variable(ob_path)"
+    "# Adds a coordinate for the spectra and TOF\n",
+    "ds.coords[\"spectrum\"] = sc.Variable([\"spectrum\"], values=np.arange(ds[\"sample\"].shape[1]))\n",
+    "stitched = sc.Dataset(coords={\"tof\": sc.Variable([\"tof\"], unit=sc.units.us,\n",
+    "                              values=np.arange(start=rebin_parameters[\"start\"], stop=rebin_parameters[\"stop\"],\n",
+    "                                               step=rebin_parameters[\"width\"], dtype=np.float64))})"
```

Comment:
Consider using `np.linspace`, avoiding the mess with the dtype? It lets you specify the bin count instead of the bin width, which I actually find more convenient in many cases.